scene/resources/2d/shape_2d.cpp

```diff
@@ -106,6 +106,7 @@ void Shape2D::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("collide_with_motion_and_get_contacts", "local_xform", "local_motion", "with_shape", "shape_xform", "shape_motion"), &Shape2D::collide_with_motion_and_get_contacts);
 	ClassDB::bind_method(D_METHOD("draw", "canvas_item", "color"), &Shape2D::draw);
 	ClassDB::bind_method(D_METHOD("get_rect"), &Shape2D::get_rect);
+  ClassDB::bind_method(D_METHOD("contains_point"), &Shape2D::contains_point);
```

Comment:
```suggestion
	ClassDB::bind_method(D_METHOD("contains_point", "point"), &Shape2D::contains_point);
```

Reply:
Went ahead and removed contains_point API bindings from Shape2D, since it exists in CollisionShape2D, and the Shape2D functions use a point relative to the origin, which isn't super useful